xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2501 @@
+          </para>
+<screen>&prompt.sudo;<command>systemctl enable tuned</command></screen>
+          <para>
+            However, this command does not starts the &tuned; process. If you check the status of
+            the systemd service now, it should be <emphasis>enabled</emphasis> and
+            <emphasis>inactive</emphasis>.
+          </para>
+        </step>
+      </procedure>
+    </sect2>
+
+    <sect2 xml:id="sec-tuning-tuned-start">
+      <title>Starting &tuned;</title>
+      <para>
+        After enabling the associated systemd service for &tuned;, for the current session you can
+        start &tuned; either as a background daemon or as a normal process connected to the TTY.
+      </para>
+      <para>
+        Alternatively, you can enable the associated systemd service and start the daemon
+        simultaneously.
+      </para>
+      <procedure>
+        <step>
+          <para>
+            To start &tuned; only for the current session, perform one of the following steps:
+          </para>
+          <itemizedlist>
+            <listitem>
+              <para>
+                Start &tuned; as a background process and let systemd manage the lifecycle of the
+                daemon:
+              </para>
+<screen>&prompt.sudo;<command>systemctl start tuned</command></screen>
+              <note>
+                <title>Profile run by &tuned;</title>
+                <para>
+                  Once activated, &tuned; starts tuning the system based on the default profile
+                  that is appropriate for the system, or the currently active profile. To change
+                  it, use the <command>tuned-adm</command> command. For more information, see the
+                  sections <xref linkend="sec-tuning-tuned-profiles"></xref> and
+                  <xref linkend="sec-tuning-tuned-plugins"></xref>.
+                </para>
+              </note>
+            </listitem>
+            <listitem>
+              <para>
+                Start &tuned; as a normal process connected to the TTY only for the current
+                session:
+              </para>
+<screen>&prompt.sudo;<command>tuned</command></screen>
+              <tip>
+                <title>Running the <command>tuned</command> command</title>
+                <para>
+                  You can use this command to activate a profile, or even run it as a daemon. For
+                  more information, run the <command>tuned --help</command> command.
+                </para>
+              </tip>
+            </listitem>
+          </itemizedlist>
+        </step>
+        <step>
+          <para>
+            <emphasis>ALTERNATIVELY</emphasis>, to enable a systemd service for future boots and
+            simultaneously start the &tuned; daemon in the current session, run the following
+            command:
+          </para>
+<screen>&prompt.sudo;<command>systemctl enable --now tuned</command></screen>
+        </step>
```

Comment:
That's should actually not a separate step, but a `stepalternative` as shown below.